src/style.css

```diff
@@ -1,3 +1,139 @@
-body {
+/* stylelint-disable-next-line no-empty-source */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* Під'єднюємо кольори і в подальшому їх використовуємо в коді */
+:root {
+  --white-color: #fff;
+  --dark-blue-color: #060b35;
+  --gray-color: #616070;
+  --light-blue-color: #00acdc;
+}
+
+/* Це говорить браузеру враховувати любі коднони
+і внутрішні відступи в значеннях */
+* {
+  box-sizing: border-box;
+}
+/* Закоментовані параметри відцентровують контент по середині дисплею */
+.page {
   margin: 0;
+  /* display: flex;
+  align-items: center;
+  justify-content: center; */
+}
+
+.card {
+  width: 200px; /*ширина*/
+  box-sizing: border-box; /*враховування кордонів відступу*/
```

Comment:
remove box-sizing because you already have it in * selector and remove all comments